src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -23,4 +23,125 @@
-"""
\ No newline at end of file
+"""
+
+helps['storage'] = """
+    type: group
+    short-summary: Durable, highly available, and massively scalable cloud storage
+"""
+helps['storage account'] = """
+    type: group
+    short-summary: Commands to manage your Storage accounts
+"""
+helps['storage account keys'] = """
+    type: group
+    short-summary: Commands to manage your Storage aaccount keys
+"""
+helps['storage blob'] = """
+    type: group
+    short-summary: REST-bsed object storage for unstructured data
+"""
+helps['storage blob copy'] = """
+    type: group
+    short-summary: Commands to manage your blob copy operations
+"""
+helps['storage blob lease'] = """
+    type: group
+    short-summary: Commands to manage leases of your storage blob
+"""
+helps['storage blob metadata'] = """
+    type: group
+    short-summary: Commands to manage your blob metadata
+"""
+helps['storage blob service-properties'] = """
+    type: group
+    short-summary: Commands to view Storage blob service properties
+"""
+helps['storage container'] = """
+    type: group
+    short-summary: Commands to manage your storage containers
+"""
+helps['storage container lease'] = """
+    type: group
+    short-summary: Commands to manage leases of your storage containers
+"""
+helps['storage container metadata'] = """
+    type: group
+    short-summary: Commands to manage your storage container metadata
+"""
+helps['storage container policy'] = """
+    type: group
+    short-summary: Commands to manage stored access policies of your storage container
+"""
+helps['storage cors'] = """
+    type: group
+    short-summary: Commands to manage your Storage Cross-Orgin Resource Sharing (CORS)
+"""
+helps['storage directory'] = """
+    type: group
+    short-summary: Commands to manage your Storage file directory
+"""
+helps['storage directory metadata'] = """
+    type: group
+    short-summary: Commands to manage your Storage file directory metadata
+"""
+helps['storage entity'] = """
+    type: group
+    short-summary: Commands to manage Storage table entities
+"""
+helps['storage file'] = """
+    type: group
+    short-summary: File shares that use the standard SMB 3.0 protocal
+"""
+helps['storage file copy'] = """
+    type: group
+    short-summary: Commands to manage your file copy operations
+"""
+helps['storage file metadata'] = """
+    type: group
+    short-summary: Commands to manage your file metadata
+"""
+helps['storage logging'] = """
+    type: group
+    short-summary: Commands to view Storage logging information
+"""
+helps['storage message'] = """
+    type: group
+    short-summary: Commands to manage Storage queue messages
+"""
+helps['storage metrics'] = """
+    type: group
+    short-summary: Commands to manage your Storage metrics properties
+"""
+helps['storage queue'] = """
+    type: group
+    short-summary: Effectively scale apps according to traffic using queues
```

Comment:
>Effectively scale apps according to traffic using queues [](start = 19, length = 56)

This could be misleading. I don't think this is the only use case for storage queues. This could use a long-summary as well that gives or at least points to documentation highlighting the differences between storage queues and service bus queues. I doubt the service team would drive that (granted we don't have service bus onboarded yet).